openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -3,7 +3,7 @@
 <div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
     <div class="openassessment problem" id="openassessment">
         <div class="wrapper--grid">
-            <h2 class="openassessment__title problem__header">{{ title }}</h2>
+            <h2 class="openassessment__title problem__header">{% trans title %}</h2>
```

Comment:
I don't see a reason to use `{% trans %}` here, since the title will be user-supplied text.  That means in general, we won't have translations for it.